rpcs3/emucore.vcxproj.filters

```diff
@@ -1410,6 +1413,8 @@
     </ClInclude>
     <ClInclude Include="Emu\RSX\overlays.h">
       <Filter>Emu\GPU\RSX</Filter>
+    <ClInclude Include="Emu\Cell\lv2\sys_overlay.h">

```

Comment:
forget again add < /ClInclude> beforer this line ^^

Reply:
Which is interesting since visual studio generated this code

Reply:
yes, without that, separater folder on Visual studio is totaly broken for that